gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Wed Jan 28 14:53:04 PST 2015
+#Sat May 09 15:41:17 PDT 2015
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.2.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.2.1-all.zip
```

Comment:
So this is a suggestion that Android Studio makes about using "distribution with sources". I think the `-all` variant includes the gradle sources as well, so you can step into the code and look at it. Probably not needed, I can revert that change if you want